README.md

```diff
@@ -1,3 +1,264 @@
-### Local Environment
-Create new virtual environment and run `pip install -r dev_requirements.txt`
-Run `pre-commit install` to initialize [pre-commit](https://pre-commit.com/)
+# Neptune Scale client
+
+> [!NOTE]
+> This package only works with the `3.0` version of neptune.ai called Neptune Scale, which is in beta.
```

Comment:
```suggestion
> This package only works with the `3.0` version of neptune.ai called Neptune Scale, which is in beta.
>
> It's supported on Linux and MacOS.
```

Reply:
Can we have a separate _Prerequisites_ section instead, listing Neptune Scale, Python 3.8+, and Linux/MacOS platforms?
Until we have the docs out, we can use this section to keep adding any other prerequisites